lasy/utils/openpmd_helper.py

```diff
@@ -88,6 +88,7 @@ def write_to_openpmd_file(
     # Store metadata needed to reconstruct the field
     m.set_attribute("angularFrequency", 2 * np.pi * c / wavelength)
     m.set_attribute("polarization", pol)
+    m.set_attribute("position", grid.position)
```

Comment:
Instead of storing this as a separate attribute, could modify how we save `m.grid_global_offset` to include this shift? This would be more consistent with the openPMD standard and would avoid having to impose changes on the way other codes read this quantity.